mapping.json

```diff
@@ -36,6 +36,7 @@
         { "regexRule": "chartbeat\\.com\\/js\\/chartbeat\\.js", "surrogate": "chartbeat.js" }
     ],
     "facebook.net": [
+         { "regexRule": "facebook\\.net\\/.*\\/all\\.js", "surrogate": "fb-sdk.js", "action": "block-ctl-fb" },
```

Comment:
Nit: Is the indentation wrong here, or is it just how GitHub is rendering the diff?